modules/system/assets/ui/js/popup.js

```diff
@@ -86,6 +87,10 @@
                 error: function(jqXHR, textStatus, errorThrown) {
                     this.error(jqXHR, textStatus, errorThrown).done(function(){
                         self.hide()
+                        if (self.isLoading) {
```

Comment:
@bennothommo I think you could just use `hideLoading()` here.

```
if (self.isLoading) {
   self.hideLoading()
}
```

Reply:
@w20k Good point - changed! :)